Fakebook.Profile/Fakebook.Profile.RestApi/ApiModel/ProfileApiModel.cs

```diff
@@ -66,6 +77,8 @@ public ProfileApiModel(DomainProfile p)
             PhoneNumber = p.PhoneNumber;
             BirthDate = p.BirthDate;
             Status = p.Status;
+            Followers = p.FollowerEmails;
+            Following = p.FollowingEmails;
```

Comment:
don't you need to map these properties in the other direction too (`ToDomainProfile`), to support updates?

Reply:
Modified the DomainProfile constructor and added the ToDomainProfile to support updates